src/Extracting/DatabaseTransactionHelpers.php

```diff
@@ -7,23 +7,27 @@
 trait DatabaseTransactionHelpers
 {
     /**
+     * @param string $connection
+     *
      * @return void
      */
-    private function startDbTransaction()
+    private function startDbTransaction(String $connection = null)
```

Comment:
Canonically, in PHP, it's `string`, not `String`. Let's keep it uniform.

Reply:
If we take the other approach for everything, we don't need this anyways :)